.github/workflows/release.yml

```diff
@@ -4,6 +4,12 @@ on:
   push:
     tags:
       - 'v*.*.*'
+
+env:
+  AZURE_FUNCTIONAPP_NAME: 'logscale_azure_function'   # set this to your function app name on Azure
+  AZURE_FUNCTIONAPP_PACKAGE_PATH: 'azure_function_timer_trigger_code'       # set this to the path to your function app project, defaults to the repository root
+  PYTHON_VERSION: '3.9'                     # set this to the python version to use (e.g. '3.6', '3.7', '3.8')
```

Comment:
We have Pyhton 3.8 in the arm.json file should we keep the same in here to install the dependencies?

Reply:
yes. done